pkgs/misc/emulators/citra/base.nix

```diff
@@ -0,0 +1,73 @@
+{ pname, version, src, branchDesc, branchName
+, lib, stdenv, config, fetchFromGitHub
+, catch2, cmake, ninja, pkg-config, wrapQtAppsHook
+, boost173, ffmpeg, libusb1, qtbase, qtdeclarative, qtmultimedia, qttools, rapidjson, SDL2
```

Comment:
Can we use an older boost version or boost17x?

Reply:
boost17x seems to build fine.

Reply:
```suggestion
, boost17x, ffmpeg, libusb1, qtbase, qtdeclarative, qtmultimedia, qttools, rapidjson, SDL2
```
then It would be nice to not track a specific boost version so that we can remove older ones eventually.